src/dynamic_boundary_conditions/hirds_depth_data_from_db.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jan 20 16:36:59 2022.
+
+@author: pkh35
+"""
+
+import sys
+import pandas as pd
+import geopandas
+from src.dynamic_boundary_conditions import hirds_depth_data_to_db
+
+
+def catchment_area_geometry_info(file):
+    """Extract geometry from the shape file, returns polygon."""
+    catchment = geopandas.read_file(file)
+    catchment = catchment.to_crs(4326)
+    catchment_area = catchment.geometry[0]
+    return catchment_area
+
+
+def get_each_site_hirds_depth_data(ari, duration, site, engine, rcp=None, time_period=None):
+    """Get hirds rainfall depth data from the database."""
+    if rcp is None and time_period is not None or rcp is not None and time_period is None:
+        print("check the arguments of get_hirds_depth_data\n if rcp is None,time period should be None and vice-versa")
+        sys.exit()
+    else:
+        if rcp is not None and time_period is not None:
+            query = f"""select site_id, "{duration}h" from hirds_rain_depth where site_id='{site}' and ari={ari} and\
+                rcp='{rcp}' and time_period='{time_period}'"""
+        else:
+            query = f"""select site_id, "{duration}h" from hirds_rain_depth where site_id='{site}' and ari={ari} and\
+                rcp is null and time_period is null"""
```

Comment:
This is a potential security concern because of SQL injection. Prepared statements should be used here, instead of formatting strings.

Reply:
This should be changed everywhere or raised as an issue

Reply:
Raised issue #24